Document/GroupManager.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Sonata\UserBundle\Document;
 
-use FOS\UserBundle\Document\GroupManager as BaseGroupManager;
+use FOS\UserBundle\Doctrine\GroupManager as BaseGroupManager;
```

Comment:
What's wrong with [this class](https://github.com/sonata-project/SonataUserBundle/blob/3.x/Document/GroupManager.php) ?